arcgis-ios-sdk-samples/Maps/Map loaded/README.md

```diff
@@ -14,7 +14,10 @@ Open the sample to load the map. The load status will be displayed on screen.
 
 ## How it works
 
-The sample uses Key-Value Observing on the `AGSMap`’s `loadStatus` property to determine when the status has changed. The `AGSLoadStatus` is `loaded` when any of the following criteria are met:
+1. Create an `AGSMap` and add it to the `AGSMapView`.
+2. Use Key-Value Observing on the `AGSMap`'s `loadStatus` property to determine when the status has changed.
+
+The `AGSLoadStatus` is `loaded` when any of the following criteria are met:
```

Comment:
The type (`AGSLoadStatus`) doesn't have a value, so this should either be:

```suggestion
The load status is `loaded` when any of the following criteria are met:
```

Or: 

```suggestion
The value of the `loadStatus` property is `loaded` when any of the following criteria are met:
```